src/create-model.jl

```diff
@@ -76,6 +75,38 @@ function construct_dataframes(graph, representative_periods, constraints_partiti
     end
     dataframes[:storage_level_inter_rp].index = 1:size(dataframes[:storage_level_inter_rp], 1)
 
+    # Dataframe to store the constraints for assets with maximum energy between (inter) representative periods
+    # Only for assets with max energy limit
+    dataframes[:max_energy_inter_rp] = DataFrame(
+        (
+            (
+                (asset = a, periods_block = periods_block) for
+                periods_block in graph[a].timeframe_partitions
+            ) for a in A if !ismissing(graph[a].max_energy_timeframe_partition)
+        ) |> Iterators.flatten,
+    )
+    if size(dataframes[:max_energy_inter_rp], 1) == 0
+        dataframes[:max_energy_inter_rp] =
+            DataFrame(; asset = Symbol[], periods_block = PeriodsBlock[])
+    end
+    dataframes[:max_energy_inter_rp].index = 1:size(dataframes[:max_energy_inter_rp], 1)
+
+    # Dataframe to store the constraints for assets with minimum energy between (inter) representative periods
+    # Only for assets with min energy limit
+    dataframes[:min_energy_inter_rp] = DataFrame(
+        (
+            (
+                (asset = a, periods_block = periods_block) for
+                periods_block in graph[a].timeframe_partitions
+            ) for a in A if !ismissing(graph[a].min_energy_timeframe_partition)
+        ) |> Iterators.flatten,
+    )
+    if size(dataframes[:min_energy_inter_rp], 1) == 0
+        dataframes[:min_energy_inter_rp] =
+            DataFrame(; asset = Symbol[], periods_block = PeriodsBlock[])
+    end
+    dataframes[:min_energy_inter_rp].index = 1:size(dataframes[:min_energy_inter_rp], 1)
```

Comment:
Just a reminder to create an issue to evaluate refactoring this section (can be after merge).